ui/app/AppLayouts/Chat/views/ChatView.qml

```diff
@@ -204,4 +215,17 @@ StatusAppThreePanelLayout {
     Component.onCompleted: {
         rootStore.groupInfoPopupComponent = groupInfoPopupComponent;
     }
+
+    ActivityCenterPopup {
```

Comment:
Wasn't this moved elsewhere recently?

Reply:
yeah, waiting for https://github.com/status-im/status-desktop/pull/6870